package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "3box": "^1.18.1",
+    "3box": "^1.20.1",
```

Comment:
3box still seems to have errors (even on their own website...). So unfortunately this isn't working yet